Gems/ROS2/Code/Source/SdfAssetBuilder/SdfAssetBuilderSettings.h

```diff
@@ -8,11 +8,32 @@
 
 #pragma once
 
+#include <AzCore/std/containers/unordered_map.h>
 #include <AssetBuilderSDK/AssetBuilderSDK.h>
 #include <AzCore/Settings/SettingsRegistry.h>
 
 namespace ROS2
 {
+    struct SdfAssetPathResolverSettings
+    {
+        public:
+            AZ_RTTI(SdfAssetPathResolverSettings, "{51EDDB99-FE82-4783-9C91-7DF403AD4EFA}");
+
+            SdfAssetPathResolverSettings() = default;
+            virtual ~SdfAssetPathResolverSettings() = default;
+
+            static void Reflect(AZ::ReflectContext* context);
+
+        using UriPrefixMap = AZStd::unordered_map<AZStd::string, AZStd::vector<AZStd::string>>;
```

Comment:
I recommend changing the value type to be a vector of `AZ::IO::Path`, since that is what is being mapped.
```suggestion
        using UriPrefixMap = AZStd::unordered_map<AZStd::string, AZStd::vector<AZ::IO::Path>>;
```